providers/websockets/kraken/messages.go

```diff
@@ -0,0 +1,338 @@
+//		"channelID": 10001,
+//		"channelName": "ticker",
+//		"event": "subscriptionStatus",
+//		"pair": "XBT/EUR",
+//		"status": "subscribed",
+//		"subscription": {
+//			"name": "ticker"
+//		}
+//	}
+//
+// Bad response:
+//
+//		{
+//			"errorMessage": "Subscription depth not supported",
+//			"event": "subscriptionStatus",
+//			"pair": "XBT/USD",
+//			"status": "error",
+//			"subscription": {
+//				"depth": 42,
+//				"name": "book"
+//			}
+//	}
+//
+// ref: https://docs.kraken.com/websockets/#message-subscriptionStatus
+type SubscribeResponseMessage struct {
+	// ChannelID is the channel ID.
+	ChannelID uint64 `json:"channelID"`
+
+	// ChannelName is the channel name.
+	ChannelName string `json:"channelName"`
+
+	// Event is the event name that is sent to the client when the server has
+	// received the subscription request.
+	Event string `json:"event"`
+
+	// Pair is the asset pair that was subscribed to.
+	Pair string `json:"pair"`
+
+	// Status is the status that is sent to the client when the server has
+	// received the subscription request.
+	Status string `json:"status"`
+
+	// Subscription is the subscription details.
+	Subscription Subscription `json:"subscription"`
+
+	// ErrorMessage is the error message that is sent to the client when the
+	// server has received the subscription request.
+	ErrorMessage string `json:"errorMessage"`
+}
+
+// TickerResponseMessage is the message that is sent to the client when the
+// server has a price update for the subscribed asset pair. This is specific
+// to the ticker subscription.
+//
+//		[
+//	  	0, 						// ChannelID
+//	  	{
+//	    	"a": [ 				// Ask array
+//	      		"5525.40000", 	// Best ask price
+//	     	 	1, 				// Whole lot volume
+//	     	 	"1.000" 		// Lot volume
+//	    	],
+//	    	"b": [ 				// Bid array
+//	      		"5525.10000", 	// Best bid price
+//	     	 	1, 				// Whole lot volume
+//	      		"1.000" 		// Lot volume
+//	    	],
+//	    	"c": [ 				// Close array
+//	      		"5525.10000", 	// Price
+//	      		"0.00398963" 	// Lot volume
+//	    	],
+//	    	"h": [ 				// High price array
+//	      		"5783.00000", 	// Today
+//	      		"5783.00000" 	// Last 24 hours
+//	    	],
+//	    	"l": [ 				// Low price array
+//	      		"5505.00000", 	// Today
+//	      		"5505.00000"	// Last 24 hours
+//	   	 	],
+//	    	"o": [				// Open price array
+//	     	 	"5760.70000", 	// Today
+//	     	 	"5763.40000"	// Last 24 hours
+//	   	 	],
+//	  	 	"p": [ 				// Volume weighted average price array <- This is the value we want
+//	   	   		"5631.44067", 	// Value Today
+//	   	   		"5653.78939" 	// Value 24h
+//	  	  	],
+//	   	 	"t": [ 				// Number of trades array
+//	   	   		11493,			// Today
+//	   	   		16267 			// Last 24 hours
+//	  	 	],
+//	   	 	"v": [				// Volume array
+//	    	  "2634.11501494", 	// Value Today
+//	    	  "3591.17907851"	// Value 24h
+//	   	 	]
+//	 	 },
+//	  	"ticker", 				// Channel name
+//	  	"XBT/USD" 				// Asset pair
+//
+// ]
```

Comment:
formatting is ok when look at github.dev not sure why it looks incorrect on the other.

Reply:
also looks fine when looking locally